redfish-core/lib/log_services.hpp

```diff
@@ -1075,8 +1075,8 @@ inline void createDumpTaskCallback(
             }
             taskData->payload->httpHeaders.emplace_back(std::move(headerLoc));
 
-            BMCWEB_LOG_CRITICAL << "INFO: " << createdObjPath.str
-                                << ": Dump creation task completed";
+            BMCWEB_LOG_INFO << "INFO: " << createdObjPath.str
```

Comment:
Leaving "INFO:" in this message on purpose?

Reply:
Done!